sessions-form.html

```diff
@@ -0,0 +1,101 @@
+---
+layout: page
+order: 0
+title: "Sessions: Confirm Participation"
+titleDisplay: "Sessions"
+---
+
+<p>In order to confirm your participation in Our Networks 2019, please complete the below form by Monday July 21, 2019 (note: we use formspree.io to send responses).</p>
+
+<hr />
+
+<form action="https://formspree.io/orga@ournetworks.ca" class="form-container" method="POST">
+  <div class="mb-2">
+    <div class="flex flex-wrap">
+      <label class="fw-bold w-100" for="session">Select your session:</label>
+      <input list="sessions" id="session" class="input mb-1 w-75" name="session" required>
+    </div>
+    <datalist id="sessions">
+      {% include forms/sessions-datalist.html %}
+    </datalist>
+    <div class="flex mb-05 flex-align-baseline">
+      <input class="mr-05" type="checkbox" id="description" name="review_confirmation" value="reviewed"><label class="fw-bold" for="description">The presentation description has been reviewed</label>
+    </div>
+  </div>
+  <div class="mb-2">
+    <h3>Participation</h3>
+    <p>Let us know if you can attend:</p>
+    <div class="flex mb-05 flex-align-baseline">
+      <input type="radio" class="mr-05" id="participation1" name="participation" value="will present" required><label class="fw-bold" for="participation1">I/we will be presenting</label>
+    </div>
+    <div class="flex mb-05 flex-align-baseline">
+      <input type="radio" class="mr-05" id="participation2" name="participation" value="no longer presenting" required><label class="fw-bold" for="participation2">I/we can no longer present</label>
+    </div>
+  </div>
+  <div class="mb-2">
+    <h3>Recording</h3>
+    <p>Let us know if we can livestream and record your presentation:</p>
+    <div class="flex mb-05 flex-align-baseline">
+      <input type="checkbox" class="mr-05" id="recording1" name="recording" value="live stream"><label class="fw-bold" for="recording1">The presentation can be live streamed</label>
+    </div>
+    <div class="flex mb-05 flex-align-baseline">
+      <input type="checkbox" class="mr-05" id="recording2" name="recording" value="record"><label class="fw-bold" for="recording2">The presentation can be recorded and posted on the Our Networks website under a Creative Commons License</label>
+    </div>
+    <small class="block mt-1">You have the right to withdraw consent for posting after the video is recorded. You can see an example of the previous recorded talks at: <a href="https://ournetworks.ca/recorded-talks/">https://ournetworks.ca/recorded-talks/</a></small>
+  </div>
+  <div class="mb-2">
+    <h3>Presenter Details</h3>
+    <p>Please confirm you preferred pronunciation and pronouns as presenter(s):</p>
+    <div class="flex">
+      <div class="w-33">
+        <label class="fw-bold" for="presenters">Presenter:</label>
```

Comment:
```suggestion
        <label class="fw-bold" for="presenters">Presenter name:</label>
```
More explicit field label